packages/diagrams/frontend/sirius-components-diagrams/src/sprotty/views/RectangleView.tsx

```diff
@@ -108,6 +111,35 @@ export class RectangleView extends RectangularNodeView {
   }
 }
 
+const debugFontSize = 6;
+
+const styleDebug = {
+  fontSize: '6px',
+  'font-family': 'Arial, Helvetica, sans-serif',
+  'font-weight': 'normal',
+  'font-style': 'normal',
+  'text-decoration': 'none',
+  'text-anchor': 'start',
+};
+
+const debugInfos = (node, size) => {
+  if (node.root.debug) {
+    return (
+      <g>
+        <text x={0} y={0 + debugFontSize * 2} style={styleDebug}>
+          x:{node.bounds.x.toFixed(2)} y:{node.bounds.y.toFixed(2)}
+        </text>
+        <text x={size.width / 2} y={size.height + debugFontSize * 2} style={styleDebug}>
+          Width: {size.width.toFixed(2)}
+        </text>
+        <text x={0} y={size.height - debugFontSize * 2} style={styleDebug}>
+          Height: {size.height.toFixed(2)}
+        </text>
+      </g>
+    );
+  }
```

Comment:
Specify an explicit return in case the debug mode is not activated with `return null`